### PR TITLE
New ui analytics

### PR DESCRIPTION
**Pull request by calellowitz**
Adding ga tracking events for locations, new nav bar, case management, auto case closure, and bulk archive
@NoahCarnahan 
FYI @amsagoff @krsdimagi

### CODE REVIEW

File: corehq/apps/app_manager/static/app_manager/js/case-config-ui-2.js [hunk 1]
```diff
@@ -492,6 +492,7 @@ var CaseConfig = (function () {
                 }, self);
 
                 self.case_properties.push(property);
+                ga_track_event('Case Management', 'User Case Management', 'Save Properties')
```
Comment (dimagimon) [inline corehq/apps/app_manager/static/app_manager/js/case-config-ui-2.js]: Missing semicolon.

File: corehq/apps/style/templates/style/includes/global_navigation_bar.html [hunk 2]
```diff
@@ -4,6 +4,35 @@
 {% load tzmigration %}
 
 {% if request.couch_user %}
+    <script type="text/javascript">
+        $('#settingsmenu-projectSettings').click(function () {
+            ga_track_event('Nav Bar', 'Click Gear Icon');
+        });
```
Comment (NoahCarnahan) [inline corehq/apps/style/templates/style/includes/global_navigation_bar.html]: I think these should all be `gaTrackLink()` calls, which will ensure that the call to the google analytics completes before we navigate away from the current page.

**Comment by calellowitz** [comment]
@NoahCarnahan, I changed all of the simple click cases to gaTrackLink, making sure to get all the ones that could change the page. I left some of the others in their existing functions, because they were leveraging existing js in the page or were not just attaching click listeners to specific buttons

**Comment by NoahCarnahan** [comment]
Awesome, thanks. I'll merge when travis finishes.

**Comment by NoahCarnahan** [comment]
build failure is possibly real?

**Comment by calellowitz** [comment]
I kicked the build again, the failure was related to static compression so I doubt it but if it fails again I will look deeper

**Comment by calellowitz** [comment]
@NoahCarnahan, found source of failing test. fixed it, this build should work

**Pull request merged**